src/account/operations/transaction/high_level/send_nft.rs

```diff
@@ -89,23 +88,11 @@ impl AccountHandle {
                         .with_nft_id(address_and_nft_id.nft_id)
                         .with_unlock_conditions(vec![UnlockCondition::Address(AddressUnlockCondition::new(address))]);
                     outputs.push(nft_builder.finish_output(token_supply)?);
-                    // Add custom input
-                    custom_inputs.push(nft_output_data.output_id);
                 }
             } else {
                 return Err(crate::Error::NftNotFoundInUnspentOutputs);
             };
         }
-        let options = match options {
-            Some(mut options) => {
-                options.custom_inputs.replace(custom_inputs);
-                Some(options)
-            }
-            None => Some(TransactionOptions {
-                custom_inputs: Some(custom_inputs),
-                ..Default::default()
-            }),
-        };
```

Comment:
Can you explain the rationale of removing this ?

Reply:
When the nft is controlled by an alias or nft, then the input nft is not enough